File: Javascript/1008.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let employeeNumber = parseInt(lines.shift());
let workTime = parseInt(lines.shift());
let paymentHour = parseFloat(lines.shift());
let salary = workTime * paymentHour;

console.log("NUMBER = " + employeeNumber);
console.log("SALARY = U$ " + salary.toFixed(2))

File: C/1008.c
#include <stdio.h>
 
int main() {
 
    int employeeNumber, workTime;
    float paymentValue;

    scanf("%d", &employeeNumber);
    scanf("%d", &workTime);
    scanf("%f", &paymentValue);

    printf("NUMBER = %d\n", employeeNumber);
    printf("SALARY = U$ %.2f\n", workTime * paymentValue);
 
    return 0;
}